Python/Product/PythonTools/PythonTools.csproj

```diff
@@ -692,6 +692,7 @@
     <Compile Include="Properties\AssemblyInfo.cs" />
     <Compile Include="PkgCmdID.cs" />
     <Compile Include="PythonTools\WebProject\PythonWebPropertyPage.cs" />
+    <Compile Include="SharedProject\Debugger\ProvideDebugAdapterAttribute.cs" />
```

Comment:
Don't add files to the SharedProject folder - it isn't real (it's actually under `Common` rather than `Python`, and all its files are linked in).

This one can go under `PythonTools\Debugger`.